### ISSUE TITLE
Suggest: Stop harvesting on start of Green Day

### ISSUE THREAD

**Issue by feeltheburn** [raduprv/Eternal-Lands#197]
"Umm, dude, WTF, today is Mother Nature's day, it's poor taste to harvest!"

If you're harvesting when Green Day starts, you continue trying to harvest, getting this message every time a harvest attempt would normally be made. 

Can lead to a flooded text.

Suggest: When this message appears, if it's possible automatically do something that would normally stop harvests.

**Comment by pjbroad** [comment]
This sounds like a server but, the client may be able to do something though.

**Comment by feeltheburn** [comment]
Yeah, was thinking that message could be used where, if it's sent, it would indicate harvesting should be stopped.

There's no reasons to keep harvesting going if that message occurs.
